helix-core.el

```diff
@@ -319,6 +328,60 @@ of the matching word in backward searches."
     (search-backward helix-current-search)
     (helix--select-region (match-beginning 0) (match-end 0))))
 
+(defun helix-find-next-char (char)
+  "Goto next CHAR."
```

Comment:
Just a consistency note: "Go to" instead of "Goto" to match the other docstrings